pkg/datapath/linux/route/route_darwin.go

```diff
@@ -9,15 +9,14 @@ import (
 	"fmt"
 
 	"github.com/vishvananda/netlink"
-
-	"github.com/cilium/cilium/pkg/mtu"
 )
 
 // errUnsupportedOp is a common error
 var errUnsupportedOp = fmt.Errorf("Route operations not supported on Darwin")
 
 // Replace is not supported on Darwin and will return an error at runtime.
-func Replace(route Route, mtuConfig mtu.Configuration) error {
+// The mtuConfig parameter is ignored as this is a stub implementation.
+func Replace(route Route, mtuConfig any) error {
```

Comment:
Just wondering, what was the motivation behind this change? Seems unrelated to the BGP change, so probably would make sense to separate it at least to an individual commit with the clarification in its description.